packages/contentstack/bin/run

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env node
-require('dotenv').config();
+require('dotenv').config({ path: '../../.env' });
```

Comment:
Do we need env to be part of the run file only dev is enough I guess.? Because this file will be used by the production build. So in that case we need to make `dotenv` as a main dependency, and that is not necessary if we don't need any .env file existing in prod/GA CLI. Also, prod CLI can't find the mentioned path.!

Reply:
@netrajpatel

Reply:
@antonyagustine Resolved.